### PR TITLE
Setup product list and create order

### PR DESCRIPTION
**Pull request by DefyCab**
### This PR-contains

- Removed enzyme library
- Create acceptance test for user to see product list
- Adds axios 
- Complete e2e specs and implement functionality
- Adds fixture for orderCreateResponse
- Small fix and both tests go green

@elvitak @gsim1903 @giacoletti @Mljungho

### CODE REVIEW

File: cypress/integration/userCanOrderProduct.feature.js [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+      .first()
+      .within(() => {
+        cy.get("button").click();
+      });
+  });
+
+  it("is expected to make a POST request", () => {
+    cy.wait("@Orders.create").its("request.method").should("eq", "POST");
+  });
+});
```
Comment (lukeperrin10) [inline cypress/integration/userCanOrderProduct.feature.js]: in the fixture file you have a message "Pizza was added to your order" not testing for this?

File: src/App.jsx [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+
+  useEffect(() => {
+    Products.index().then((data) => {
+      setProducts(data);
+    });
+  }, []);
+
+  const addToOrder = (product_id) => {
+    Orders.create(product_id, 1).then((response) => {
+      debugger;
```
Comment (lukeperrin10) [inline src/App.jsx]: Don't need to say too much about this one!

**Review by lukeperrin10 — CHANGES_REQUESTED** [review]
Debugger has got to go and I need a clarification on the test spec for can order a product

File: cypress/integration/userCanOrderProduct.feature.js (hunk 1 above)
Comment (DefyCab) [reply cypress/integration/userCanOrderProduct.feature.js]: Either test for it or don't have it in the fixture?

File: src/App.jsx (hunk 2 above)
Comment (DefyCab) [reply src/App.jsx]: It's gone!

File: cypress/integration/userCanOrderProduct.feature.js (hunk 1 above)
Comment (giacoletti) [reply cypress/integration/userCanOrderProduct.feature.js]: We need to write a test for that.

Comment (DefyCab) [reply cypress/integration/userCanOrderProduct.feature.js]: I removed the fixture. When we have written the test we re-add it.

File: cypress/fixtures/products.json [hunk 3]
```diff
@@ -0,0 +1,25 @@
...
+        },
+        {
+            "id": 3,
+            "name": "Risotto",
+            "price": 20.00,
+            "created_at": "2021-12-07",
+            "updated_at": "2021-12-07"
+        }
+    ]
+}
```
Comment (tochman) [inline cypress/fixtures/products.json:25]: Please mind code formatting

**Review by lukeperrin10 — APPROVED** [review]
Removing the message from the fixture file wasn't really what I was looking for. You will likely use that message in your pop up that you presented during your LoFi's. I will merge this so you can carry on but have a think about what I said.

File: cypress/fixtures/products.json (hunk 3 above)
Comment (tochman) [reply cypress/fixtures/products.json:25]: Also, please rename this file to `productsIndexResponse`

**Pull request merged**